v3p/zlib/zconf.h

```diff
@@ -151,7 +145,7 @@
 #    define FAR far
 #  endif
 #endif
-#if defined(__BORLANDC__) && (defined(__SMALL__) || defined(__MEDIUM__))
+#if defined(__MEDIUM__))
```

Comment:
Arrg.  I've removed that code at least 2 times already.  I must have put the fix in a different patch set.  I'll remove here again.